File: Atividades/ATP21/Conta.java
package Atividades.ATP21;

public class Conta {
    private double saldo;
    private int codigoCliente;
 
    public void setSaldo(double saldo) {
        this.saldo = saldo;
    }
    public double getSaldo() {
        return saldo;
    }
    public void setCodigoCliente(int codigoCliente) {
        this.codigoCliente = codigoCliente;
    }
    public int getCodigoCliente() {
        return codigoCliente;
    }

    public void transferencia (double valorTranf){
        saldo -= valorTranf;
        System.out.printf("\n O valor tranferido: R$ %.2f",valorTranf );
    }

}

File: ACAJNode/Aula007/model/Conta.js
class Conta{
    #numero;
    #agencia;
    #saldo;

    get numero(){
        return this.#numero;
    }
    set numero(numero){
        this.#numero = numero;
    }
    get agencia(){
        return this.#agencia;
    }
    set agencia(agencia){
        this.#agencia = agencia;
    }
    get saldo(){
        return this.#saldo;
    }
    set saldo(saldo){
        this.#saldo = saldo;
    }

}
export default Conta;